docs/en/observability/apm/act-on-data/index.asciidoc

```diff
@@ -0,0 +1,28 @@
+////
+TO DO:
+Does this framing make sense?
+////
```

Comment:
It does, but will this page go away if we move alerts and links to "Incident management?"

Reply:
[Here's what I did](https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/apm-act-on-data.html) for now. Take a look at https://github.com/elastic/observability-docs/pull/4230#discussion_r1761594950 and let me know what you think about moving the custom links page.

Reply:
Good point about being specific to APM. I didn't realize that everything else in the incident management section was more widely applicable. I like your proposal!